katello-agent/katello-agent.spec

```diff
@@ -88,13 +88,21 @@ cp extra/katello-agent-send.cron %{buildroot}%{_sysconfdir}/cron.d/%{name}
 rm -rf %{buildroot}
 
 %post
-chkconfig goferd on
 %if 0%{?fedora} > 18 || 0%{?rhel} > 6
-  /bin/systemctl start goferd > /dev/null 2>&1 || :
+  systemctl enable goferd
+  if [systemctl status goferd] then
```

Comment:
Brackets aren't needed, and this actually won't work, [ and [[ expect a condition.  I'd use something closer to my suggestion, especially with redirecting output to /dev/null.